Source/WebKit/WebProcess/GPU/graphics/RemoteImageBufferProxy.h

```diff
@@ -231,6 +236,7 @@ class RemoteImageBufferProxy : public WebCore::ConcreteImageBuffer<BackendType>
     void clearBackend() final
     {
         setNeedsFlush(false);
+        prepareForBackingStoreChange();
         didFlush(m_sentFlushIdentifier);
```

Comment:
`prepareForBackingStoreChange()` below asserts `ASSERT(!hasPendingFlush())`. But `didFlush()` sets `m_receivedFlushIdentifier` to `m_sentFlushIdentifier`. This actually will guarantee `hasPendingFlush()` will return false. I think you may need to call `prepareForBackingStoreChange()` after calling `didFlush()`.

Reply:
Good catch, done